language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/ExtractToFunctionCodeAction.java

```diff
@@ -490,6 +501,15 @@ private Optional<ArgListsHolder> getArgLists(CodeActionContext context, List<Sym
         return Optional.of(argListsHolder);
     }
 
+    private Optional<TypeSymbol> getTypeSymbol(CodeActionContext context, TypeSymbol typeSymbol,
```

Comment:
Better if we can rename this method or add a comment because here we are using a type symbol to get another type symbol.